### DISCUSSION TITLE
general/otp

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#54] [Announcements]
# general/otp

필자가 이해한 내용을 최대한 간단하게 써내려가는 글로, 부족한 부분이나 틀린 내용이 있을 수 있습니다.

피드백은 언제나 환영합니다!

스마트폰이 나오기 전, 어느 사이트에 가입할 때 비밀번호를 까먹을 것을 대비하여 '초등학교 때 가장 기억에 남는 선생님 성...

https://www.howdy-mj.me/general/otp

**Comment by JuneParkCode** [comment]
덕분에 놓치고 있던 부분에 대해서 잘 짚고 넘어갈 수 있었습니다. 좋은 글 감사합니다.
